src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -594,12 +594,14 @@ private IObjectOrCollectionInitializerOperation CreateBoundCollectionInitializer
 
         private IOperation CreateBoundObjectInitializerMemberOperation(BoundObjectInitializerMember boundObjectInitializerMember)
         {
-            Lazy<IOperation> instance = new Lazy<IOperation>(() => new InstanceReferenceExpression(
-                    semanticModel: _semanticModel,
-                    syntax: boundObjectInitializerMember.Syntax,
-                    type: boundObjectInitializerMember.ReceiverType,
-                    constantValue: default(Optional<object>),
-                    isImplicit: true));
+            Lazy<IOperation> instance = new Lazy<IOperation>(() => boundObjectInitializerMember.MemberSymbol.IsStatic ?
+                                                                   null :
```

Comment:
>null [](start = 67, length = 4)

Another opportunity to use singleton.

Reply:
Done.

---
In reply to: [149578172](https://github.com/dotnet/roslyn/pull/22972#discussion_r149578172) [](ancestors = 149578172)